serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,165 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
+:context: serverless-ossm-tls
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+.Prerequisites
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install xref:../../service_mesh/service_mesh_install/preparing-ossm-installation.adoc#preparing-ossm-installation[{ProductName}].
+* In this example, `openssl` is used to generate certificates, but you can use any certificate generation tool you like to create these.
+
+.Procedure
+
+. Create TLS certificates:
++
```

Comment:
These commands are not typical for creating certificates trusted by CA. Maybe we should use examples more widely used?

Reply:
I used this example because Istio uses it https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#generate-client-and-server-certificates-and-keys. So it would be alright as it is.